tools/publish

```diff
@@ -70,6 +70,8 @@ npm version $VERSION_ARGS || exit 1
 
 git push && git push --tags || exit 1
 
+npm install
```

Comment:
I'm not sure this is necessary if using `npx`, but seems reasonable 👍🏼

Reply:
👍🏼